src/main.rs

```diff
@@ -26,13 +16,66 @@ const EXTRA_HELP: &str = "\
 
 type IoResult<T> = Result<T, Box<dyn std::error::Error>>;
 
-fn flag_set(flag: &str, matches: &ArgMatches) -> bool {
-    matches!(
-        matches.value_source(flag),
-        Some(ValueSource::CommandLine | ValueSource::EnvVariable)
+#[macro_export]
+macro_rules! version_0 {
+    () => {
+        concat!("v", crate_version!())
+    };
+}
+
+#[macro_export]
+macro_rules! version_info {
+    () => {
+        concat!(crate_name!(), " ", $crate::version_0!())
+    };
+}
+
+const fn about_info() -> &'static str {
+    concat!(
+        version_info!(),
+        "\n",
+        crate_authors!(),
+        "\nEnsure Cargo.toml dependency tables are sorted.",
     )
 }
 
+#[derive(clap::Parser, Debug)]
+#[command(author = crate_authors!(), version = version_0!(), about = about_info(), bin_name = "cargo sort", after_help = EXTRA_HELP)]
```

Comment:
What is the default author and version? Is that not good enough?

Reply:
This cause output this:
```
cargo-sort v2.0.1
```
and the default output is `cargo-sort 2.0.1` there no `v`.

Reply:
The version number can also be seen in `--help`, which I think is a more user-friendly feature.

Reply:
We don't have it right now, nobody is asking for it and it requires extra code to be written. In my opinion, that is a clear "we shouldn't do it".

I think the overall refactoring makes sense why I'd leave it at that: A true refactoring shouldn't change functionality.

Reply:
modified.